homeassistant/components/ecovacs/vacuum.py

```diff
@@ -1,6 +1,17 @@
 """Support for Ecovacs Ecovacs Vaccums."""
 import logging
 
+from sucks import (
```

Comment:
I worry that these imports are too generically-named for a top-level import like this.

Perhaps just import sucks, and then use `sucks.Spot`, `sucks.Stop` in the appropriate methods?

Reply:
Sure thing 👍